src/OpenTelemetry.AutoInstrumentation/Tagging/InstrumentationTags.cs

```diff
@@ -18,9 +18,9 @@ namespace OpenTelemetry.AutoInstrumentation.Tagging;
 
 internal abstract class InstrumentationTags : TagsList
 {
-    protected static readonly IProperty<string>[] InstrumentationTagsProperties =
+    protected static readonly IProperty<string?>[] InstrumentationTagsProperties =
```

Comment:
seems a good candidate for refactor. If we leave properties / tags out that have `null` value, then there will be less calls that do nothing.

Reply:
Like the idea, but I think it will be to much for this PR #1862